src/Common/src/Interop/Windows/mincore/Interop.GetFullPathNameW.cs

```diff
@@ -22,19 +22,25 @@ partial class mincore
         [DllImport(Libraries.CoreFile_L1, EntryPoint = "GetFullPathNameW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false)]
         private static extern int GetFullPathNamePrivate(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero);
 
-        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero)
+        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer)
```

Comment:
My biggest concern is exposing direct access to the P/Invoke and unwrapped usages slipping through.

Isn't normalizing access to the APIs legitimate? I would think that we would be trying to enforce proper behavior at this level. For example: Why do we allow you to specify a StringBuffer and a size? It would prevent a lot of issues to only expose StringBuffer and get the Capacity in the wrappers.

There is also a concern about discovery.

I think of PathInternal as "interop" as well. There isn't anything in there that doesn't also directly apply to interacting with the Win32 APIs.